app/src/main/java/com/findmyrecycling/AppModule.kt

```diff
@@ -0,0 +1,11 @@
+package com.findmyrecycling
+
+import com.findmyrecycling.service.ProductService
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+@JvmField
+val appModule = module {
+    viewModel { MainViewModel(get()) }
+    single <ProductService>{ ProductService() }
+}
```

Comment:
Added AppModule.kt file for Koin based on these references: https://kotlinlang.org/docs/java-to-kotlin-interop.html#instance-fields
https://blog.mindorks.com/kotlin-koin-tutorial